src/components/notification-detail.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import { Card, CardContent, Stack, FormLabel } from '@mui/material';
+import { useTranslator } from '../hooks';
+import { Scheduler } from '../handler';
+import { LabeledValue } from '../components/labeled-value';
+
+interface INotificationItemProps {
+  label: string;
+  value: string | boolean;
+}
+
+interface INotificationDetailsProps {
+  notification: Scheduler.INotification;
+}
+
+const NotificationItem: React.FC<INotificationItemProps> = ({
+  label,
+  value
+}) => {
+  const displayValue =
+    typeof value === 'boolean' ? (value ? 'Yes' : 'No') : value;
+
+  return (
+    <LabeledValue
+      label={label}
+      value={displayValue}
+      style={{ flex: '1 1 100%' }}
+    />
+  );
+};
+
+export const NotificationDetails: React.FC<INotificationDetailsProps> = ({
+  notification
+}) => {
+  const trans = useTranslator('jupyterlab');
+
+  return (
+    <Card>
+      <CardContent>
+        <FormLabel component="legend" sx={{ mb: 2 }}>
+          {trans.__('Notification')}
+        </FormLabel>
+        <Stack spacing={2}>
+          <FormLabel component="legend">{trans.__('Send To')}</FormLabel>
+          {notification.send_to.map((email, idx) => (
+            <NotificationItem
+              key={idx}
+              label={trans.__(`Send To ${idx + 1}`)}
+              value={email}
+            />
+          ))}
+          <FormLabel component="legend">{trans.__('Events')}</FormLabel>
+          {notification.events.map((event, idx) => (
+            <NotificationItem
+              key={idx}
+              label={trans.__(`Event ${idx + 1}`)}
+              value={event}
+            />
+          ))}
```

Comment:
The UI here needs work. For the emails, you should probably use a table, list of badges, or something else that doesn't take up a block element for every email. This gets hard to read after maybe 5 emails.

For the events, recall that the number of events is bounded to 3 presently. So you can just comma-join all the events and use a single field to indicate that.

Reply:
Considered different options. Decided to go with comma-separated string of values to display both send to symbols and notification events as it looks most consistent with how other values are displayed in details view:
![Screenshot 2023-11-02 at 9 27 53 AM](https://github.com/jupyter-server/jupyter-scheduler/assets/26686070/d370c7f0-5126-422c-b1bb-90a34eabe0a1)

In my opinion, if we for example use chips to display send to symbols (or send to and events), send to field becomes visually and logically more important than other fields. This is not always the case so always visually emphasizing send to field decreases readability:
![Screenshot 2023-11-02 at 9 25 29 AM](https://github.com/jupyter-server/jupyter-scheduler/assets/26686070/eb9b44b2-b0f0-450a-83ea-a89376a84376)
![Screenshot 2023-11-02 at 9 21 45 AM](https://github.com/jupyter-server/jupyter-scheduler/assets/26686070/956379c0-88f6-417e-bc97-e3b665752feb)


Overall, I agree that details view design could use further improvement but it needs to be redesigned as a whole to maintain visual consistency.